torchmetrics/utilities/data.py

```diff
@@ -16,6 +16,16 @@
 import torch
 from torch import Tensor, tensor
 
+from torchmetrics.utilities.imports import _TORCH_GREATER_EQUAL_1_7, _TORCH_GREATER_EQUAL_1_8
+
+if _TORCH_GREATER_EQUAL_1_8:
+    deterministic = torch.are_deterministic_algorithms_enabled
+elif _TORCH_GREATER_EQUAL_1_7:
+    deterministic = torch.is_deterministic
+else:
```

Comment:
```suggestion
elif _TORCH_GREATER_EQUAL_1_7:
    deterministic = torch.is_deterministic
elif _TORCH_GREATER_EQUAL_1_6:
    deterministic = torch._is_deterministic
else:
```
v1.6 supports that https://github.com/pytorch/pytorch/blob/v1.6.0/torch/__init__.py#L322